roomranger/src/main/java/org/launchcode/roomranger/data/RoomAssignedRepository.java

```diff
@@ -0,0 +1,9 @@
+package org.launchcode.roomranger.data;
```

Comment:
I think you can delete all the assigned room data, I think Francesca will go to build them.